jscomp/repl.js

```diff
@@ -44,11 +44,11 @@ function prepare() {
     e(`./bin/jsgen.exe --`)
     e(`./bin/jscmj.exe`)
 
-    e(`ocamlc.opt -w -30-40 -no-check-prims -I bin -I outcome_printer ext.cma outcome_printer.cma bin/config_whole_compiler.mli bin/config_whole_compiler.ml bin/js_compiler.mli bin/js_compiler.ml -o jsc.byte`)
+    e(`ocamlc.opt -w -30-40 -no-check-prims -I bin -I ../lib -I outcome_printer ext.cma outcome_printer.cma ../lib/config_whole_compiler.mli ../lib/config_whole_compiler.ml bin/js_compiler.mli bin/js_compiler.ml -o jsc.byte`)
 
     e(`rm -rf  ${playground}/pre_load.js`)
     e(`cp ./pre_load.js ${playground}`)
-    e(`cp ../lib/amdjs/*.js ${playground}/stdlib`)
+    e(`cp ../lib/es6/*.js ${playground}/stdlib`)
```

Comment:
this would only work on very few browsers? can we delay such change

Reply:
Oops, this is unintended. I was experimenting because I got an empty amdjs directory. But I forgot to revert this change.